docs/plugins-distribute.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Package and distribute plugins
+description: Share your custom plugins so that others can use them.
+---
+
+There is no "official" way to distribute plugins with MyST. However, you can use common workflows in the JavaScript ecosystem to distribute your plugins so that others can use them. This page documents a few approaches that may work.
+
+## Package plugins into a single ESM file 
+
+JavaScript uses the [ECMAScript Modules standard](https://nodejs.org/api/esm.html) for packaging and distributing scripts. Your plugins can be bundled and distributed in the same way.
+
+## Use a builder to build an ESM package
+
+There are several "builders" in the JavaScript ecosystem that make it easy to quickly bundle JavaScript in distributable packages. The resulting artifact can then be shared directly, or published to a package registry like [NPM](https://npmjs.com).
+
+We recommend [using `esbuild`](https://esbuild.github.io/) to accomplish this.
```

Comment:
```suggestion
We recommend [`esbuild`](https://esbuild.github.io/).
```

Our docs have a lot of words to read through, so as a general principle I'd embrace conciseness.